src/main/java/com/fatec/springapi4/entity/Partner.java

```diff
@@ -1,30 +1,28 @@
-package com.fatec.springapi4.entity;
+package com.fatec.springapi4
+.entity;
```

Comment:
faltou remover essa quebra de linha
```suggestion
package com.fatec.springapi4.entity;
```

esse overlook também está presente nos arquivos `/entity/Qualifier.java` e `/entity/Track.java`